src/components/themeMediaPlayer.js

```diff
@@ -27,6 +27,7 @@ function playThemeMedia(items, ownerId) {
         }).then(function () {
             currentOwnerId = ownerId;
         });
+        document.getElementsByClassName("nowPlayingBarTop")[0].style.display = "none";
```

Comment:
tbh, looks a bit hacky.
1. Doesn't this break `nowPlayingBar` for normal music playback after touching theme song?
  - Go to item page with theme song.
  - After theme song started, go to the music library and start normal music playback.
2. TVs (the real ones - with `browser.tv === true`) don't have `nowPlayingBar`, iirc. So this will probably fail to access to `undefined` element.

I think we need to tell `nowPlayingBar` somehow that we don't want it to be displayed. Then hide it like this:
https://github.com/jellyfin/jellyfin-web/blob/4502c408f1656eb1948adc88b8b1eb96d207c336/src/components/nowPlayingBar/nowPlayingBar.js#L667-L670

P.S.: we use single quotes.

Reply:
Yes, it's a bit hacky. Any idea of how to do it properly?

Reply:
I changed mentioned block like this
```js
        if (!state.NowPlayingItem || layoutManager.tv || !state.IsFullscreen) {
            hideNowPlayingBar();
            return;
        }
```
`state.IsFullscreen` property replicates `fullscreen` in options for `playbackManager.play`. By default it is `true` and only in `themeMediaPlayer` it is set to `false`. Video player uses `fullscreen` to choose between normal mode and backdrop mode (animated background). So it can probably be used to hide `nowPlayingBar` (similar to "backdrop mode").
I am still not totally sure, but it works and everything is done in `nowPlayingBar`.

Reply:
That's a way better solution, thank you so much.